beacon-chain/powchain/service.go

```diff
@@ -616,6 +616,9 @@ func (s *Service) processBlockHeader(header *gethTypes.Header) {
 // batchRequestHeaders requests the block range specified in the arguments. Instead of requesting
 // each block in one call, it batches all requests into a single rpc call.
 func (s *Service) batchRequestHeaders(startBlock, endBlock uint64) ([]*gethTypes.Header, error) {
+	if startBlock > endBlock {
```

Comment:
I agree with you and @nisdas here. Sorry for suggesting otherwise! I suppose a batch request of 1 block is allowed.